columnflow/tasks/cutflow.py

```diff
@@ -217,11 +231,21 @@ def get_point(mask=Ellipsis):
                 mask = True
                 for step in steps:
                     if step not in arr.steps.fields:
-                        raise ValueError(
-                            f"step '{step}' is not defined by selector {self.selector}",
-                        )
-                    # incrementally update the mask and fill the point
-                    mask = mask & arr.steps[step]
+                        if self.missing_selector_step_strategy == "error":
+                            raise ValueError(
+                                f"step '{step}' is not defined by selector {self.selector}",
+                            )
+                        elif self.missing_selector_step_strategy == "skip":
+                            continue
+                        elif self.missing_selector_step_strategy == "dummy":
+                            pass
+                        else:
+                            assert False
```

Comment:
Just a suggestion for de-nesting. Also, I skipped the AssertionError since the choice parameter already does a value validation. I think we can rely on this and don't need to raise here.

```suggestion
                        if self.missing_selector_step_strategy == "error":
                            raise ValueError(
                                f"step '{step}' is not defined by selector {self.selector}",
                            )
                        if self.missing_selector_step_strategy == "skip":
                            continue
```